cfme/tests/containers/test_containers_summary.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import StatusBox
+from utils import testgen
+from utils.version import current_version
+import time
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+
+# container_dashboard
+# CMP-9822
+def test_containers_summary_containers(provider):
+    """ Containers overview page > Containers widget > Containers summary
+       This test checks that the amount of containers in the system is shown correctly
+        in the Containers widget in the
+       Overview menu
+       Steps:
+           * Goes to Compute --> Containers --> Overview
+           * Checks how many Containers are shown in the Containers Widget
+           * Goes to Containers summary page and checks how many Containers are shown there.
+           * Checks the amount is equal
+       """
+    sel.force_navigate('container_dashboard')
```

Comment:
When this test is run on PRT, it will return 0 for the values, because you didn't include the setup_provider fixture. Without it, you will get the provider fixture, but the provider won't be added to CFME, you also may want the fixture to ```has_no_container_providers```